### DISCUSSION TITLE
Title Instead of Image on Beamer Titlepage

### DISCUSSION THREAD

**Discussion by timbmg** [tudace/tuda_latex_templates#392] [Q&A]
Is it possible to put the title (+ authors and affiliation) in the vertical center of the title page? AFAIK, this is very common in the beamer template. But in the TUDA template, if you do not have a suitable image, the title slide looks empty.

I am looking for something along these lines:

![image](https://user-images.githubusercontent.com/11020443/183373873-1cf1f30e-540f-4560-a300-b2edbd900cac.png)

**Comment by TeXhackse** [comment]
The TUDa Design guideline doesn't allow to modify the title position. In care there is no suitable image it requires a colored box there. I just realized that tuda-ci doesn't support that by default (yet). 

You can archieve this by setting:

```
\setbeamercolor*{titlegraphic}{fg=accentcolor}
\titlegraphic{\rule{\width}{\height}}
```

To disable the colored block on top you can set the option `colortitle=false` and to add the author information to the title itesel `authorontitle=true`.

I will add the change of the colordefinition if `\titlegraphic` is empt. Therefore the suggestion will be compatible to the change.

**Comment by schoeps** [comment]
Wouldn't it be easier to add a nice image? Many colleagues do this because it allows you to give a more personalized start. In my group we either use an image representing the research covered in the talk (e.g. some lab picture) or our building (e.g. in your case I assume https://www.informatik.tu-darmstadt.de/media/ukp/pixabay_images/headers/tu_darmstadt_piloty-gebaeude_1300x0.jpg). If this does not work, you can still use TU Darmstadt press pictures (https://www.tu-darmstadt.de/universitaet/aktuelles_meldungen/medienservice/pressebilder_campus_und_gebaeude/pressebilder_detailseite_15552.de.jsp).
